asab/storage/elasticsearch.py

```diff
@@ -52,8 +52,10 @@ def __init__(self, app, service_name, config_section_name='asab:storage'):
 
 		self.URL = Config.get(config_section_name, 'elasticsearch_url')
 		parsed_url = urllib.parse.urlparse(self.URL)
+		url_path = parsed_url.path if parsed_url.path != "" else "/"
```

Comment:
what happens if parsed_url.path is not empty string ("") but it still does not end with trailing slash?